### DISCUSSION TITLE
아이템 업로드시 오류

### DISCUSSION THREAD

**Discussion by colta21** [naverz/zepeto-studio-kor#302] [Issue Report (Legacy)]
![image](https://user-images.githubusercontent.com/98699000/151704314-3e4f6b7f-3e57-412f-883b-94c67efd1d39.png)

제페토 파일 컨버팅후에 아이템 업로드하는데 계속 이런 오류가 떠서 막히고 있네요 혹시 방법아시는분 있으실까요?

**Comment by runasy-koonta** [comment]
https://studio.zepeto.me/kr/guides/creating-your-item-2d-designers

이 페이지에 산출물은 반드시 png여야 한다고 나와있는데, 혹시 gif로 내보내진 산출물을 컨버팅시에 포함시키신건 아니신가요?

**Reply by colta21** [reply]
아하 업로드 제페토 파일에 gif 파일을 지웠더니 해결되었습니다 감사합니다.
